app/preact/src/client/preview/render.tsx

```diff
@@ -1,8 +1,9 @@
 import { h, render } from 'preact';
```

Comment:
I found that `h` is needed in `preact` to wrap transpiled `jsx`, but DeepScan doesn't like it. It seems like there is a way to resolve this by using webpack plugins, but is it necessary?